src/Aspire.Dashboard/Resources/ControlsStrings.resx

```diff
@@ -386,6 +389,18 @@
   <data name="FluentDataGridHeaderCellResizeButtonText" xml:space="preserve">
     <value>Resize</value>
   </data>
+  <data name="FluentDataGridHeaderCellGrowAriaLabelText" xml:space="preserve">
```

Comment:
nit, why add "Text" on the end of some of these? looks like fluent itself does not, and Column width above does not

Reply:
It's a pseudo-pattern we've used a decent amount, but we could standardize names